bootstrap/registration/registry.go

```diff
@@ -68,6 +72,10 @@ func createRegistryClient(
 		}
 	}
 
+	if len(bootstrapConfig.Registry.Host) == 0 || bootstrapConfig.Registry.Port == 0 || len(bootstrapConfig.Registry.Type) == 0 {
+		return nil, errors.New("Registry config is incomplete, missing common config? Use -cp or -cc flags for common config.")
+	}
```

Comment:
We don't need both checks. Since this one covers all missing, the other one can be removed.

Reply:
ok, i'll update the error message slightly to include both cases.